src/GitVersionCore/Helpers/ExtensionMethods.cs

```diff
@@ -1,57 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Text.RegularExpressions;
-
-namespace GitVersion.Helpers
-{
-    internal static class ExtensionMethods
```

Comment:
Was this file moved? Seems like Git didn't detect it properly.

Reply:
It was removed, and the members were placed under the correct `*Extensions` classes